root.c

```diff
@@ -2006,6 +2177,10 @@ static void io_thread(struct unlocked_watchman_root *unlocked)
     w_pending_coll_unlock(&unlocked->root->pending);
 
     if (!pinged && w_pending_coll_size(&pending) == 0) {
+#ifndef _WIN32
+      process_pending_symlink_targets(unlocked->root);
```

Comment:
prefer to pass `&unlocked` here instead, so that it is clear what the locking requirements are for `process_pending_symlink_targets`